utils/meta.go

```diff
@@ -25,9 +25,6 @@ func NewValue(t reflect.Type) (v reflect.Value) {
 func ToArray(value interface{}) (values []string) {
 	switch value := value.(type) {
 	case []string:
-		if len(value) == 1 && value[0] == "" {
```

Comment:
You are right. The return becomes a nil slice which I did not see in Go Playground's output. How about replacing those lines with values = []string{} (https://play.golang.org/p/ANIBdnO3go), which I have just submitted for pull request?